jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -39,7 +39,7 @@
           (ros::set-param (format nil "~a_hand/state" (send *ri* :arm-symbol2str arm)) (symbol2str state)))
         (:initialize
           (ros::ros-info "[main] ~a, ~a" arm state)
-          (setq tote-contents (send *ri* :get-tote-contents))
+          (setq tote-contents (get-tote-contents))
```

Comment:
In my understanding, @wkentaro thinks "robot interface" is the system including all programs for robot hardware to do a certain work. I agree with this and with this thought, `*ri*` can have `:get-tote-contents`. Because "robot interface" have to know tote contents for stow task.